infra/event/test/TestClaimableResource.cpp

```diff
@@ -123,9 +137,12 @@ TEST_F(TestClaimableResource, ClaimUrgentTakesPrecedence)
 
 TEST_F(TestClaimableResource, TwoConsecutiveReleasesBeforeReleaseOfClaimIsProcessedResultsInNoAdditionalClaimsGranted)
 {
-    claimerA.Claim([this]() { claimerA.GrantedClaim(); });
```

Comment:
So true single-line lambdas are not possible anymore?

Reply:
It is when we set a ColumnLimit.